src/wheatley.ts

```diff
@@ -263,8 +263,6 @@ export class Wheatley {
 
     database: WheatleyDatabaseProxy;
 
-    link_blacklist: any;
```

Comment:
This is actually used and important

Reply:
Ah I see what you did elsewhere, that might be ok

Reply:
Yeah I thought I'd clean that up while I'm at it. We should probably come up with some systematic way to handle inter-component dependencies.